tests/integ/test_tf_script_mode.py

```diff
@@ -47,6 +47,7 @@ def test_mnist(sagemaker_session, instance_type):
                            sagemaker_session=sagemaker_session,
                            py_version='py3',
                            framework_version=TensorFlow.LATEST_VERSION,
+                           metric_definitions=[{'Name': 'train:global_steps', 'Regex': 'global_step\/sec:\s(.*)'}], # noqa
```

Comment:
does flake no longer enforce two spaces before the "#" on an inline comment? also, what's the `noqa` for?

Reply:
flake8 doesn't like the \s and \/

Reply:
does it go away if you do `r'global_step\/sec:\s(.*)'`?

Reply:
let me try that